pyproject.toml

```diff
@@ -6,10 +6,20 @@ readme = "README.md"
 authors = []
 requires-python = ">=3.13"
 dependencies = [
+    "flax>=0.10.2",
+    "hilbertcurve>=2",
     "imageio[ffmpeg]>=2.37.0",
-    "jax[cuda12]>=0.4.38",
+    "jax>=0.5.0 ; sys_platform != 'linux'",
+    "jax[cuda12]>=0.5.0 ; sys_platform == 'linux'",
+    "matplotlib>=3.10.0",
     "mujoco>=3.2.6",
     "mujoco-mjx>=3.2.6",
+    "numpy>=2.2.2",
+    "numpy-stl>=3.2.0",
+    "pytest>=8.3.4",
```

Comment:
Are all these dependencies needed for users to run the project? pytest is already a developer dependency, and we don't need sphinx (that's a documentation library). Not sure about the others.